challenge-03/README.md

```diff
@@ -150,8 +175,34 @@ deve conter no retorno da frase acima é "metro" no lugar de "metros".
 método), que será concatenada com a frase de retorno, mostrando a resposta
 correta, de acordo com os dados inseridos no objeto.
 */
-?
+
 
 // Agora, apresente-se ;)
-?
-```
+pessoa.apresentacao = function(){
+  var s;
+  var i;
+  var m;
+  
+  if(pessoa.sexo === "Feminino"){
+    var s = 'a';
+    }
+  else{
+    var s = 'o';
+    }
+    
+  if(pessoa.idade === 1 ){
+    var i = 'ano';
+    }
+  else{
+    var i = 'anos';
+    }
+  
+  if(pessoa.caminhouQuantosMetros === 1){
+    var m = 'metro';
+    }
+    else{
+    var m = 'metros'
+  }
+return 'Olá, eu sou '+s+' '+pessoa.nome+' '+pessoa.sobrenome+', tenho '+pessoa.idade+' '+i+', '+pessoa.altura+', meu peso é '+pessoa.peso+' e, só hoje, eu já caminhei '+pessoa.caminhouQuantosMetros+' '+m+'!';
+  
+}
```

Comment:
Procure verificar também os espaçamentos e indentação dessa função. Se você perceber, os `else` não são necessários. Você pode declarar as variáveis somente uma vez no início da função, com os valores padrão, e usar os `if` para modificar os valores das variáveis já declaradas, diminuindo bastante o código e deixando ele bem mais legível ;)